File: Materal.ConDep/Materal.ConDep.Web/MateralConDepWeb/src/app/services/models/app/appListModel.ts
/**
 * 应用列表
 */
export interface AppListModel {
    /**
     * 唯一标识
     */
    ID: string;
    /**
     * APP状态
     */
    AppStatus: number;
    /**
     * APP状态文本
     */
    AppStatusText: string;
    /**
     * 名称
     */
    Name: string;
    /**
     * 路径
     */
    AppPath: string;
    /**
     * 主模组名称
     */
    MainModuleName: string;
    /**
     * 运行参数
     */
    Parameters: string;
}

File: Level3/Materal.ConDep/Materal.ConDep.Services/Models/AppListModel.cs
using Materal.Common;
using Materal.ConDep.Services.Enums;
using System;

namespace Materal.ConDep.Services.Models
{
    /// <summary>
    /// 应用列表
    /// </summary>
    public class AppListModel
    {
        /// <summary>
        /// 唯一标识
        /// </summary>
        public Guid ID { get; set; }
        /// <summary>
        /// APP状态
        /// </summary>
        public AppStatusEnum AppStatus { get; set; }
        /// <summary>
        /// APP状态文本
        /// </summary>
        public string AppStatusText => AppStatus.GetDescription();
        /// <summary>
        /// 名称
        /// </summary>
        public string Name { get; set; }
        /// <summary>
        /// 路径
        /// </summary>
        public string AppPath { get; set; }
        /// <summary>
        /// 主模组名称
        /// </summary>
        public string MainModuleName { get; set; }
        /// <summary>
        /// 运行参数
        /// </summary>
        public string Parameters { get; set; }
    }
}
